### PR TITLE
Add email lowercaser validation with JS

### PR DESCRIPTION
**Pull request by alexansaa**
Validation has already been implemented in other merge requests with HTML. In this milestone the next features are implemented to improve contact form validation:
- Implementing lowercase string validation on the email input value, with JavaScript.
- Contact form "submit" button class changed from "submit" to "button" for catching on-click events on JS, preventing the form from submitting by default.

### CODE REVIEW

**Review by Eternalgratis — CHANGES_REQUESTED** [review]
# Status: Invalid code ❌❌❌
## Hi @alexansaa , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request❌

## You have submitted a project, that didn't show pair programming. This milestone is meant to be a collaboration. Kindly ensure your partner contributes to this milestone and you can submit whenever you are ready.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -124,8 +125,22 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
...
+                    <br>
+                    <br>
+                    <br>
+                    <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" tabindex="23">
+                    <hr class="InputSeparation">
+                    <textarea name="comment" id="comment" cols="30" rows="5" class="text_comment" placeholder="Write some text here..." tabindex="24"></textarea>
+                    
+                </form>-->
+                <!--<div>Get in touch</div>
+                <button id="SubmitButton" class="msjButton" tabindex="25" onclick="validateForm()">Get in touch</button>-->
```
Comment (jerryowusu) [inline index.html]: Kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. Please consider removing 
all the commented codes. :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi Team

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost-paul-walker


## Highlights
Validation set up :+1: 
Nice design :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
Validations working :+1: 
Nice code :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**